Library/ViewModels/PaymentSourceSelected.swift

```diff
@@ -1,33 +1,26 @@
 import Foundation
-public enum PaymentSourceSelected: Equatable {
-  case savedCreditCard(String)
-  case setupIntentClientSecret(String)
-  case paymentIntentClientSecret(String)
 
-  public var savedCreditCardId: String? {
-    switch self {
-    case let .savedCreditCard(value):
-      return value
-    default:
-      return nil
-    }
-  }
+/* PledgePaymentMethodsViewController used to return multiple types of selected payments.
+ Now it can only return stored credit cards. Leaving this enum here in case we ever add
+ more payment types in the future.
+ */
+
+public typealias StripePaymentMethodID = String
+public typealias KSRCreditCardId = String
```

Comment:
Just trying to make things clearer than `case savedCreditCard(String, String?)` which is what it really is.